CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+- Fixed: Remove unused define environment variables in webpack's config
```

Comment:
It's not really a fix, it's a boilerplate update, can you prefix with "## Boilerplate" and change the message to "- Removed: useless injected env variable to client build" ?